frame/staking/src/pallet/mod.rs

```diff
@@ -140,6 +141,9 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxNominatorRewardedPerValidator: Get<u32>;
 
+		/// Something that can provide a sorted list of voters is a somewhat sorted way.
```

Comment:
this should have more doc IMO: it should say what is the order used for, and it should link to implemenation by `pallet-bags-list::Pallet` (AFAIK doc can link to item available in dev-dependencies) and `UseNominatorMap`.